examples/bin/worker

```diff
@@ -14,6 +14,7 @@ worker.register_workflow(AsyncHelloWorldWorkflow)
 worker.register_workflow(BranchingWorkflow)
 worker.register_workflow(CancellingTimerWorkflow)
 worker.register_workflow(CheckWorkflow)
+worker.register_workflow(ComplexResultsWorkflow)
```

Comment:
@robholland I think you're missing these new workflows and activities from the PR

Reply:
Oops! Added now.